Utils/StringUtils.cpp

```diff
@@ -98,4 +98,12 @@ bool StringUtils::DoesEmptyJSON(const std::string data)
 		return true;
 
 	return false;
-}
\ No newline at end of file
+}
+
+std::wstring StringUtils::ConvertToWideString(const std::string data)
+{
+	std::wstring wideString(data.length(), ' ');
```

Comment:
We have better solution for this: using altstr.h header.
This is part of large range of header Microsoft created for COM programming.

See example here:
https://github.com/AvivShabtay/Stresser/blob/d7bb59d3af088bfc3591f99105ac282933e8314e/Stresser/EndpointController.cpp#L20

and convert it to your use case (from std::string to std::wstring)